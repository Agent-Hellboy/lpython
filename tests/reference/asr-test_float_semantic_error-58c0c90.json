{
    "basename": "asr-test_float_semantic_error-58c0c90",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_float_semantic_error.py",
    "infile_hash": "10929e3991a4aee1a2de473fc5f8caa48b6dcf4a35e2329a15ea5f1f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_float_semantic_error-58c0c90.stderr",
    "stderr_hash": "efb6fd4cc943c01f652c604e2134ad76d5ad390e1f6ba96a52266160",
    "returncode": 2
}