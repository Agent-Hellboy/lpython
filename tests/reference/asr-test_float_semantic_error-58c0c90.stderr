semantic error: Unsupported type annotation: float
 --> tests/errors/test_float_semantic_error.py:4:9
  |
4 |     pi: float = 3.14
  |         ^^^^^ 
